webapp/playbooks/src/components/sidebar/sidebar.tsx

```diff
@@ -51,11 +51,17 @@ const Sidebar = (props: SidebarProps) => {
                     delay={OVERLAY_DELAY}
                     shouldUpdatePosition={true}
                     overlay={
-                        <Tooltip id='team-name__tooltip'>{team?.description?.length ? team.description : 'No team is selected'}</Tooltip>
+                        <Tooltip id='team-name__tooltip'>
+                            {
+                                team?.description?.length ? team.description : 'No team is selected' // eslint-disable-line formatjs/no-literal-string-in-jsx
```

Comment:
Could you file a bug to note that these aren't translatable?

Reply:
added the translations here

Reply:
That works too